src/reuse/modules/util/formatter.ts

```diff
@@ -162,7 +162,7 @@ export class Formatter {
       hour = `0${hour}`;
     }
 
-    if (dd < 10) {
+    if (dd < 10 && format !== DateFormats.MONTH_DAY_YEAR_COMMA_SHORT) {
```

Comment:
Should lowercase format before this check, the test fails otherwise.

Reply:
right, will do that